gcloud-java-examples/src/main/java/com/google/gcloud/examples/StorageExample.java

```diff
@@ -142,12 +144,12 @@ public void run(Storage storage, BlobInfo... blobInfos) {
       if (blobInfos.length == 1) {
```

Comment:
I agree actually. I just didn't want to modify too much the code. But I am happy to change!